lib/vtt.js

```diff
@@ -1469,4 +1478,4 @@
 
   global.WebVTT = WebVTT;
 
-}(this));
+}(this, (this.vttjs || {})));
```

Comment:
Actually, not certain that this is needed in this file, but it's a way to pass in the 'vttjs' object or an empty object. For example, in node, we don't need `this.vttjs` and the `browser.js` file isn't being used, so, we just want an empty object.